documentation.html

```diff
@@ -301,16 +302,159 @@ <h2 class="component-sub-heading">Examples of Badges</h2>
+             </div>
+             <div class="description-container">
+                <p class="card-text">This breakthrough book from James Clear is the most comprehensive guide on how to change your habits and get 1% better every day.A supremely practical and useful book.</p>
+             </div>
+             <div class="link-container">
+               <button class="btn btn-secondary">Buy now</button>
+               <button class="btn btn-secondary">Bookmark</button>
+             </div>
+          </div>
+        </div>
+        <iframe
+        loading="lazy"
+        src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%250A%2520%253Cdiv%2520class%253D%2522card%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522image-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522image%2522%2520src%253D%2522...%2522%2520alt%253D%2522book%2522%253E%250A%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522text-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522card-title%2522%253EAtomic%2520Habits%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch4%2520class%253D%2522card-subtitle%2522%253Eby%2520James%2520Clear%253C%252Fh4%253E%250A%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522description-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522card-text%2522%253EThis%2520breakthrough%2520book%2520from%2520James%2520Clear%2520is%2520the%2520most%2520comprehensive%2520guide%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520on%2520how%2520to%2520change%2520your%2520habits%2520and%2520get%25201%2525%2520better%2520every%2520day.A%2520supremely%2520practical%2520and%2520useful%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520book.%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522link-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253EBuy%2520now%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253EBookmark%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%250A"
+        style="width: 1024px; height: 614px; border:0; transform: scale(1); overflow:hidden;"
+        sandbox="allow-scripts allow-same-origin">
```

Comment:
The inline styling for carbon could me moved to css file

Reply:
Okay, will work on it 👍 .Thanks for the review @rdthetha